tests/copyFiles.test.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable max-len */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const { faker } = require('@faker-js/faker');
+
+const { exec } = require('child_process');
+const util = require('util');
+const execAsync = util.promisify(exec);
+
+describe('File Copy', () => {
+  const baseCommand = 'node src/app.js';
+  let sourceContent = '';
+  const tempDir = path.join('tests', faker.word.noun());
+
+  const sourceFile = path.join(tempDir, faker.system.commonFileName('txt'));
+  const destinationFile = path.join(
+    tempDir,
+    faker.system.commonFileName('txt'),
+  );
```

Comment:
What's the reason to have some constants inside the `describe` but the other's outside?

Reply:
I based on the imports system which should be out of describe, but I read that it is commonly good practise to store requires out of describe